### PR TITLE
Add a Wasm build for .NET 8

### PR DESCRIPTION
**Pull request by bricelam**
Part of dotnet/efcore#29000

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -105,7 +105,8 @@ jobs:
         config:
           # Don't forget to add a download-artifact steps in test and publish
         - '{ "tfm": "net6.0", "emsdk": "2.0.23" }' # eol: 2024-11-12
-        - '{ "tfm": "net7.0", "emsdk": "3.1.12" }' # eol: 2023-05-08
+        - '{ "tfm": "net7.0", "emsdk": "3.1.12" }' # eol: 2024-05-14
+        - '{ "tfm": "net8.0", "emsdk": "3.1.34" }' # eol: 2026-11-14
```
Comment (bricelam) [inline .github/workflows/build.yml:109]: I'm following up with the team to make sure this is still the right approach, and to see how likely it is that this number will change between now an GA.

**Comment by bricelam** [comment]
(Still waiting to hear back from the .NET Wasm team. They're in the middle of finalizing some decisions for .NET 8.)

**Comment by bricelam** [comment]
I still haven't heard back from them, but we can probably move forward with this PR. I'll send additional PRs if the version changes before GA. And if they require significant changes for how things work in .NET 8 ...well, we'll cross that bridge when/if we get there.

**Comment by ericsink** [comment]
Sounds good.  I'll move forward with this, and the math change, and whatever other pending things I'm forgetting at the moment.

**Comment by bricelam** [comment]
The only other one on my radar is the TFM warnings.

**Comment by bricelam** [comment]
Let me know if I can help with any of the work. Always happy to investigate and submit PRs

**Pull request merged**